docs/core/deploying/native-aot/libraries/ios-like-native-libraries-with-nativeaot.md

```diff
@@ -8,9 +8,12 @@ ms.date: 11/11/2024
 
 # Building native libraries for iOS-like platforms
 
-Starting from .NET 9, Native AOT officially supports publishing .NET class libraries for iOS-like platforms.
+Starting from .NET 9, Native AOT officially supports publishing .NET class libraries that don't depend on iOS workloads for iOS-like platforms.
```

Comment:
```suggestion
Starting from .NET 9, Native AOT supports publishing .NET class libraries that don't depend on .NET MAUI SDK.
```
We have separate workloads for Maui iOS and Maui Mac Catalyst. Should we just say .NET MAUI SDK?

Also, it is redundant to say "officially supported" in the official docs. When we say "supported" in the official docs, it implies official support.